tools/src/output.rs

```diff
@@ -372,11 +372,21 @@ pub fn generate_panel(
                     } else {
                         String::new()
                     };
+                    let tag = if !item.link.is_empty() {
+                        "a"
+                    } else {
+                        "span"
```

Comment:
Seems like it should probably be a button rather than a span? I guess no big deal if it makes styling too annoying but..